group_vars/satellite/satellite_config_manifest.yml

```diff
@@ -0,0 +1,5 @@
+---
+# Variables to upload manifest to Satellite (redhat.satellite.manifest)
+satellite_manifest_path: "~/manifest.zip"
+satellite_manifest_download: true
+satellite_manifest_uuid: "51ee7560-e8b3-441e-b266-f2817db5b39c"
```

Comment:
Should be removed

Reply:
not sure if we should remove it for now, it is just an example which we used during the project to be able to deploy.